map.css

```diff
@@ -81,7 +87,7 @@ a:hover {
 
 #features {
     top: 0;
-    height: 180px;
+    height: 280px;
```

Comment:
There is a problem with the feature box (upper right): the buttons/links aren't placed at the bottom anymore, but somewhere in between with a large white margin below. This looks weird. Maybe it's based on this change?

Reply:
really? looked just fine for me. I had to make the box bigger in order to make sure that all fit in

Reply:
schau mal, dass du das Fenster so anpasst, dass es bei dir funktioniert und alles sichtbar ist ohne Slider. Es muss etwas größer werden, wegen dem zusätzlichen Text. Es ist ja einfach programmiert, daher kann es mit unterschiedlichen Format nicht so gut umgehen... danke

Reply:
Can you commit the latest changes and push to this branch? I'll try and resolve this based on the latest changes.

Reply:
this should be fixed now:
```
#buttons {
    margin-left: 15px;
    position: absolute;
    bottom: 10px;
}
```
Can you confirm?

Reply:
Yes, the buttons are at the bottom again now. However, the box is just too large. But that's because it shouldn't have an absolute size. I'll see whether I can fix that.